cypress/objects/taskPage.js

```diff
@@ -11,10 +11,10 @@ class Tasks {
     getFieldTextArea() {
         return cy.get('.field__textarea');
     }
-    getValue() {
+    getBillToClient() {
```

Comment:
You could add "InputField" at the end, because the way it is now it's difficult to tell whether this is an input field, a label, a dropdown etc. I'd go for "getBillToClientInputField"